accessibility-checker.php

```diff
@@ -467,9 +490,9 @@ function edac_anww_update_post_meta() {
 
 	$option_name = 'edac_anww_update_post_meta';
 
-	if ( get_option( $option_name ) === false && EDAC_ANWW_ACTIVE ) {
+	if ( ! get_option( $option_name ) && EDAC_ANWW_ACTIVE ) {
 		update_option( $option_name, true );
-	} elseif ( get_option( $option_name ) === true && ! EDAC_ANWW_ACTIVE ) {
+	} elseif ( get_option( $option_name ) && ! EDAC_ANWW_ACTIVE ) {
```

Comment:
The `edac_anww_update_post_meta` option has a boolean value, so this condition was simplified